<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinksList</class>
 <widget class="QDialog" name="LinksList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LINKS list</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qHostMonitor.qrc">
    <normaloff>:/img/hostMonitor.png</normaloff>:/img/hostMonitor.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frameTest">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblTestName">
        <property name="text">
         <string>Test name:</string>
        </property>
        <property name="buddy">
         <cstring>editTestName</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="editTestName">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblTestMethod">
        <property name="text">
         <string>Test method:</string>
        </property>
        <property name="buddy">
         <cstring>editTestMethod</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editTestMethod">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblParentFolder">
        <property name="text">
         <string>Parent folder:</string>
        </property>
        <property name="buddy">
         <cstring>editParentFolder</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="editParentFolder">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>There is %1 link to the test, located in following folder</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListWidget" name="listLinks">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="selectionRectVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameButtons">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRemoveLinks">
        <property name="text">
         <string>Remove marked links</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRemoveTest">
        <property name="text">
         <string>Remove test and links</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qHostMonitor.qrc"/>
  <include location="../qHostMonitor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>LinksList</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
